.github/workflows/main.yml

```diff
@@ -89,18 +89,47 @@ jobs:
         echo nofiles=ignore>> "%GITHUB_OUTPUT%"
         echo moredef=-DLOVE_EXTRA_DLLS=%CD%\angle\libEGL.dll;%CD%\angle\libGLESv2.dll>> "%GITHUB_OUTPUT%"
         exit /b 0
+    - name: Download Windows SDK Setup 10.0.20348
```

Comment:
There are a lot of extra steps here, I'm not confident we could guarantee support for this in the future.

(That's not necessarily a reason to avoid doing it, but it might not last forever if we do it.)

Reply:
While it's true, unfortunately debugging tools for Windows is not installed in GitHub Actions. One way to avoid it entirely is to write `srctool -r` and `pdbstr` tool equivalent in other languages.